specification/containerinstance/resource-manager/Microsoft.ContainerInstance/preview/2024-11-01-preview/containerInstance.json

```diff
@@ -2379,11 +2387,23 @@
             "identityAcls": {
               "description": "The access control levels of the identities.",
               "$ref": "#/definitions/IdentityAcls"
+            },
+            "containerGroupProfile": {
+              "description": "The reference container group profile properties.",
+              "$ref": "#/definitions/ContainerGroupProfileReferenceDefinition"
+            },
+            "standbyPoolProfile": {
+              "description": "The reference standby pool profile properties.",
+              "$ref": "#/definitions/StandbyPoolProfileDefinition"
```

Comment:
for next reviewer, this is backfilling from 2024-05-01-preview api versions, that is missed in 2024-09-01-preview and 2024-10-01-preview